docs/change-log/index.md

```diff
@@ -202,6 +218,13 @@ The `--ensure-project` flag of the `mlrun project` CLI command is deprecated and
 | ML-3446 | Fix: Failed MLRun Nuclio deploy needs better error messages. [View in Git](https://github.com/mlrun/mlrun/pull/3241). |
 | ML-3482 | Fixed model-monitoring incompatibility issue with mlrun client running v1.1.x and a server running v1.2.x. [View in Git](https://github.com/mlrun/mlrun/pull/3180). |
 
+## v1.2.3
+
+### Closed issues
+
+| ID   | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3287 | UI now resets the cache upon MLRun upgrades, and the Projects page displays correctly.  [View in Git](https://github.com/mlrun/ui/pull/1612). |
```

Comment:
This is just ui, nothing on backend - thus should be removed.

what fixed on 1.2.2:

1. Fix presenting and serving large-sized projects

what fixed on 1.2.3:

1. Fix UI glitch presenting project membership screen
2. Optimized `/projects` endpoint to work faster
3. Reduce too verbose logs on backend side

Reply:
@liranbg 3287 was opened by a customer - therefore is included